### DISCUSSION TITLE
🐞 Paired Number is earsed on computer reboot.

### DISCUSSION THREAD

**Discussion by HaimenToshi** [aldinokemal/go-whatsapp-web-multidevice#242] [Announcements]
### What happened?

Whenever I restart my PC that runs go-whatsapp-web-multidevice, I have to re-pair my phone with this server using the QR code again. I reboot my PC once every 20 hours or so.
The server is launched through a shell script that gets executed automatically from the user's .bashrc file.

### Reproduction Steps

```bash
1. Pair your phone with this server as usual
2. Reboot
3.
...
```

### Application Version

v4.17.0

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
Which system are you using? Docker, Windows, or Linux?
